scalding-core/src/main/scala/com/twitter/package.scala

```diff
@@ -73,4 +73,6 @@ package object scalding {
       !filter.accept(p)
     }
   }
+
+  val state = new java.util.concurrent.ConcurrentHashMap[String, String]()
```

Comment:
0) this looks really scary and architecturally bad news. I hope we can find some other way.

1) this should be `private[scalding]` at the very least.

2) can we add plenty of comments here as to what the use case is here.